apps/android/app/src/main/java/chat/simplex/app/views/usersettings/NotificationsSettingsView.kt

```diff
@@ -0,0 +1,268 @@
+      Modifier.padding(start = 16.dp, end = 16.dp, bottom = 24.dp),
+      style = MaterialTheme.typography.h1
+    )
+    SectionView(null) {
+      Column(
+        Modifier.padding(horizontal = 8.dp)
+      ) {
+        SectionItemViewSpaceBetween({ showPage(CurrentPage.NOTIFICATIONS_MODE) }, padding = PaddingValues()) {
+          Text(stringResource(R.string.settings_notifications_mode_title))
+          Spacer(Modifier.padding(horizontal = 10.dp))
+          Text(
+            modes.first { it.first == notificationsMode.value }.second,
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+            color = HighOrLowlight
+          )
+        }
+        Spacer(Modifier.padding(horizontal = 4.dp))
+        SectionItemViewSpaceBetween({ showPage(CurrentPage.NOTIFICATION_PREVIEW_MODE) }, padding = PaddingValues()) {
+          Text(stringResource(R.string.settings_notification_preview_mode_title))
+          Spacer(Modifier.padding(horizontal = 10.dp))
+          Text(
+            previewModes.first { it.first == notificationPreviewMode.value }.second,
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+            color = HighOrLowlight
+          )
+        }
+      }
+    }
+  }
+}
+
+@Composable
+fun NotificationsModeView(
+  notificationsMode: State<NotificationsMode>,
+  onNotificationsModeSelected: (NotificationsMode) -> Unit,
+) {
+  val modes = remember { notificationModes() }
+
+  Column(
+    Modifier.fillMaxWidth(),
+    horizontalAlignment = Alignment.Start,
+  ) {
+    Text(
+      stringResource(R.string.settings_notifications_mode_title).lowercase().capitalize(Locale.current),
+      Modifier.padding(start = 16.dp, end = 16.dp, bottom = 24.dp),
+      style = MaterialTheme.typography.h1
+    )
+    SectionView(null) {
+      LazyColumn(
+        Modifier.padding(horizontal = 8.dp)
+      ) {
+        items(modes.size) { index ->
+          val item = modes[index]
+          val onClick = {
+            onNotificationsModeSelected(item.first)
+          }
+          SectionItemViewSpaceBetween(onClick, padding = PaddingValues()) {
+            Text(item.second)
+            if (notificationsMode.value == item.first) {
+              Icon(Icons.Outlined.Check, item.second, tint = HighOrLowlight)
+            }
+          }
+          Spacer(Modifier.padding(horizontal = 4.dp))
+        }
+      }
+    }
+    SectionTextFooter(modes.first { it.first == notificationsMode.value }.third)
+  }
+}
+
+@Composable
+fun NotificationPreviewView(
+  notificationPreviewMode: State<NotificationPreviewMode>,
+  onNotificationPreviewModeSelected: (NotificationPreviewMode) -> Unit,
+) {
+  val previewModes = remember { notificationPreviewModes() }
+
+  Column(
+    Modifier.fillMaxWidth(),
+    horizontalAlignment = Alignment.Start,
+  ) {
+    Text(
+      stringResource(R.string.settings_notification_preview_mode_title).lowercase().capitalize(Locale.current),
```

Comment:
```suggestion
      stringResource(R.string.settings_notification_preview_mode_title).lowercase().capitalize(Locale.current),
```

Use separate key for this view's title and change title to "Notification preview" ("Notifications"?) so that the context that this is in regard to notifications is not lost when this view is shown? Right now it's "Show preview"

Reply:
done